bcgov/mds
# Mds 3942 minespace submit incident notification

# Main

- Provide the functionality to create new incidents on minespace. (The button to record a new incident has to be hidden when the code review will be done)

# Other
- Changes in styling were added on antd-overrides.scss and webpack.parts.js

# How to test
- Click on ![image](https://user-images.githubusercontent.com/10526131/136301250-37b5719b-9a15-4683-8fe2-4b76320184d5.png)
- Fill the required fields on the first-step form and click on Next: 
![image](https://user-images.githubusercontent.com/10526131/136589472-4f5f3344-c4db-42fd-88d7-8abb64ce2a77.png)
- Fill the required fields on the second-step form:  
![image](https://user-images.githubusercontent.com/10526131/136589620-4a23ac2d-0ddb-4c17-bd86-64d915905630.png)
- At the bottom of the form appears an alert with a legend to check understanding of contacting an inspector:
![image](https://user-images.githubusercontent.com/10526131/136589860-e52214a9-d980-42eb-9e69-e34379ff1d52.png)
- When the user clicks on the checkbox, then the Submit button is enabled to create the new incident. 
![image](https://user-images.githubusercontent.com/10526131/136590037-cc528787-09e1-4cf0-9a8c-9591e2864890.png)
- The newly created incident is added at the top of the Incidents table:
![image](https://user-images.githubusercontent.com/10526131/136590155-4a50e224-d20d-4398-b0f4-83aaf605fbf5.png)

- Email notification example:
![image](https://user-images.githubusercontent.com/10526131/136969271-8672d182-f9df-4cae-b1ae-a0263c3dc9fc.png)




# Notes
https://bcmines.atlassian.net/browse/MDS-3942
